tests/logging_ut/minimal/minimal.c

```diff
@@ -27,7 +29,16 @@ int main(void)
     SetLastError(ERROR_SUCCESS);
 
     LogLastError("Hello World from LogLastError, everything is fine now! here's a value: 0x%x", '3');
-    
+
+#else
+    errno = EACCES;
+
+    LogLastError("Hello World from LogLastError, some access was denied! here's a value: 0x%x", '3');
```

Comment:
>LogLastError [](http://example.com/codeflow?start=4&length=12)

I am somewhat confused. Isn't this being run on Linux? Is there LogLastError on Linux?